test/browser.js

```diff
@@ -21,13 +21,13 @@
 // TODO - this interferes with render test by adding background color
 // require('tap-browser-color')();
 
-const test = require('tape');
+const test = require("tape");
 
 test.onFinish(window.browserTestDriver_finish);
 test.onFailure(window.browserTestDriver_fail);
 
-test('Browser tests', t => {
-  require('./modules');
-  require('./render');
+test("Browser tests", t => {
```

Comment:
Why? Our linter rules have always been to use single quotes. Hardly something we want to change across a huge code base?

Reply:
Ah I failed to catch this. It was done by prettier before I added the template rules. I need to open a PR for all lint changes anyways.